content/docs/user-guide/basic-concepts/output.md

```diff
@@ -4,5 +4,5 @@ match: [output, outputs]
 ---
 
 A file or directory tracked by DVC, recorded in the `outs` section of a
-DVC-file. Outputs are usually the result of stages. See `dvc add`, `dvc run`,
-`dvc import`, et al. A.k.a. _data artifact_
+`dvc.yaml` or `.dvc` file. Outputs are usually the result of stages. See
```

Comment:
Link both "`dvc.yaml`" and "`.dvc` file" to the user guide.

Reply:
Please

Reply:
Yes, they are upcoming in my next PR. All the linking has been done in basic-concepts/

Reply:
Cool. All basic concepts are done.